src/pages/health-declaration/health-declaration.ts

```diff
@@ -27,30 +26,8 @@ export class HealthDeclarationPage {
     this.signaturePadOptions = configService.getSignaturePadOptions();
   }
 
-  ngAfterViewInit() {
-    // this.signaturePad is now available
-    this.signaturePad.set('minWidth', 1); // set szimek/signature_pad options at runtime
-    this.signaturePad.clear(); // invoke functions from szimek/signature_pad API
-  }
-
-  drawComplete() {
-    this.signature = this.signaturePad.toDataURL();
-    this.validation();
-  }
-
-  clearSignaturePad() {
-    this.signaturePad.clear();
-    this.signature = null;
-  }
-
-  drawStart() {}
-
-  ionViewDidLoad() {
-    this.signaturePad.resizeCanvas();
-  }
-
   validation() {
-    if (this.confirmation && this.signature) {
+    if (this.confirmation && this.signaturePad.getSignature()) {
```

Comment:
Would be cleaner if you just returned the negated execution of the expression in the `if` statement.
But it's really just preference so it's up to you if you want to change it.

Reply:
Nice spot - Have updated